host-controller/src/main/java/org/jboss/as/host/controller/ManagedServer.java

```diff
@@ -940,4 +942,8 @@ public boolean execute(ManagedServer server) throws Exception {
         return result;
     }
 
+    protected PathAddress getAddress(){
```

Comment:
Sure, thanks! it can be changed, I will do it and I will pass the tests on local.